test/controllers/match.test.ts

```diff
@@ -0,0 +1,197 @@
+import request from 'supertest'
+import connection from '../db-helper'
+import app from '../../src/app'
+
+process.on('uncaughtException', function (err) {
+  console.log('oh no', err);
+});
+
+const creatingSkills = async () => {
+  await request(app).post('/skills/create').send({ name: 'javascript' })
+  await request(app).post('/skills/create').send({ name: 'reactjs' })
+  await request(app).post('/skills/create').send({ name: 'typescript' })
+  await request(app).post('/skills/create').send({ name: 'react' })
+  await request(app).post('/skills/create').send({ name: 'nodejs' })
+  await request(app).post('/skills/create').send({ name: 'docker' })
+  await request(app).post('/skills/create').send({ name: 'aws' })
+  await request(app).post('/skills/create').send({ name: 'gcp' })
+  await request(app).post('/skills/create').send({ name: 'iac' })
```

Comment:
You should refactor this. First make a skills object that contain all the skills:
`const skills = {
  { name: 'react' },
  { name: 'react' }
}
`
Then map through it to send the request:
`skills.map( skill => await request(app).send(skill) )`